docs/Getting-Started.md

```diff
@@ -242,6 +242,8 @@ As discussed previously, Fastify offers a solid encapsulation model, to help you
 ### Validate your data
 Data validation is extremely important and a core concept of the framework.<br>
 To validate incoming requests, Fastify uses [JSON Schema](https://json-schema.org/).
+(JTD schemas are loosely supported, but `jsonShorthand` must be disabled first)
```

Comment:
As JTD is getting popular and which will be fully supported in v4, an example of the use of it would be nice. Don't you think @Eomm ?

Reply:
I think JTD stuff should be contained in a dedicated PR and we should focus on the `jsonShorthand` flag only here